test/WebJobs.Extensions.Tests/Extensions/MobileApps/MobileTableEndToEndTests.cs

```diff
@@ -242,6 +244,10 @@ private class MobileTableEndToEndFunctions
                 [MobileTable] out TodoItem[] arrayPoco,
                 [MobileTable] IAsyncCollector<TodoItem> asyncCollectorPoco,
                 [MobileTable] ICollector<TodoItem> collectorPoco,
+                [MobileTable(TableName = tableName)] out object newObject, // we explicitly allow object
+                [MobileTable(TableName = tableName)] out object[] arrayObject,
```

Comment:
Good catch. I wanted to fail if they did that, but I wasn't. I've added that check and a new test.

---
In reply to: [64283568](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/101#discussion_r64283568) [](ancestors = 64283568)